csun-metalab/nomi
# NAM-469 -- UI: Refactor the convertDays method for courseListItem & courseBanner component

# Description
Refactored the convertTime and convertDays method into a mixin which will be called per component.
  
# Testing
1. Ensure no console errors appear when visiting course and student.
2. Make sure the proper formatted date and time appear on the course info.
